golem_messages/message/concents.py

```diff
@@ -147,6 +147,67 @@ class FileInfo(datastructures.FrozenDict):
         }
 
 
+class ForceGetTaskResult(base.Message):
+    TYPE = CONCENT_MSG_BASE + 6
+
+    __slots__ = [
+        'report_computed_task',
+    ] + base.Message.__slots__
+
+    def deserialize_slot(self, key, value):
+        value = super().deserialize_slot(key, value)
+        return tasks.deserialize_report_computed_task(key, value)
+
+
+class ForceGetTaskResultAck(base.Message):
+    TYPE = CONCENT_MSG_BASE + 7
+
```

Comment:
According to the documentation (Dokumentacja analityczna integracji Golem-Concent) a field is missing: there should be also a mandatory MessageForceGetTaskResult.